tests/0.2.4.1/enabled_with_contexts.yaml

```diff
@@ -83,7 +83,7 @@ tests:
             user:
               IsHuman: verified # different case than in feature flag rule on purpose
         expected:
-          value: true
+          value: false
```

Comment:
This was an implementation behavior in the Java client that is user-friendly but has a performance penalty and doesn't behave the same way in other clients. We're standardizing on strictness for now

Reply:
it looks like this and the case below are the same thing and only one should remain?

Reply:
or switch one of them to have the correct case and return true as a laser-targetted test of case sensitivity

Reply:
the rule is for `user.isHuman`